main.c

```diff
@@ -132,7 +132,29 @@ static void reset_devices(void) {
     #endif
 }
 
-STATIC void start_mp(supervisor_allocation *heap) {
+#if MICROPY_ENABLE_PYSTACK
+STATIC supervisor_allocation __attribute__ ((noinline)) * alloc_pystack(void) {
```

Comment:
Do you need `noinline? I'm not sure why it is needed on other functions.

Reply:
If it's not required then remove it.
It tells the compiler not to inline this function, which is something you usually don't care about.

Reply:
I removed it.